zstacklib/zstacklib/utils/iproute.py

```
@@ -160,18 +177,21 @@ def __init__(self, chunk):
         self.table = chunk['table']
         self.scope = _get_scope_name(chunk['scope'])
         self.chunk = chunk
+
     def get_related_addresses(self, namespace=None):
         '''
             :return IpAddr[]
         '''
         return query_addresses(index=self.devIndex, namespace=namespace)
```

Pylint (3.3.7)

[error] Instance of 'IpRoute' has no 'devIndex' member

(E1101)